site/content/api/Tenant-API.md

```diff
@@ -222,42 +238,41 @@ The table below provides an overview of the standard members defined for the JSO
 | Name                     | Mandatory | Type       | Default Value | Description |
 | :------------------------| :-------: | :--------- | :------------ | :---------- |
 | *tenant-id*              | *yes*     | *string*   |               | The ID of the tenant. |
-| *enabled*                | *no*      | *boolean*  | *true*        | If set to *false* the tenant is currently disabled.|
-| *adapterConfiguration*   | *yes*     | *array*    |               | A list of configuration options valid for certain adapters only. See [Adapter Configuration Format]({{< relref "#adapter-configuration-format" >}}) for details. **NB** This array must contain at least one element - an empty array is handled as an error. **NB** Only one entry with a matching adapter-id is tolerated. If multiple configurations for the same adapter-id are present it is handled as an error.|
+| *enabled*                | *no*      | *boolean*  | `true`        | If set to `false` the tenant is currently disabled.|
+| *adapters*   | *yes*     | *array*    |               | A list of configuration options valid for certain adapters only. See [Adapter Configuration Format]({{< relref "#adapter-configuration-format" >}}) for details. **NB** This array must contain at least one element - an empty array is handled as an error. **NB** Only a single entry per *type* is allowed. If multiple entries for the same type are present it is handled as an error.|
```

Comment:
Why is an empty array an error?

FMPOV this would mean that this tenant has no specific adapter configurations - all should be default.
Alternatively, the adapters array can be made optional with the same semantic.
But to force a non empty array would mean :  tenants that always need or want the default values have to be created with at least one of the adapters that have the defaults set inside - this does not make sense for me.
WDYT?

Reply:
I agree to your point. To keep the api simple I made the array optional and allowed it to be empty.